website/tutorial/part_1/03_installation.md

```diff
@@ -11,68 +11,32 @@
 
 Exosphere is not just a simple application,
 but a framework for large-scale software development of micro-service based applications.
-Developing applications consisting of many different code bases requires:
-* a __package management system__ for installing/updating the various pieces of platforms and tools as needed.
+Developing applications consisting of many different code bases requires
+a __package management system__ for installing/updating the various pieces of platforms and tools as needed.
   You don't want to get into the business of having to install several programming languages,
   frameworks, and other dependencies of your polyglot applications manually.
 
-The installation instructions below include
-making these infrastructure components available on your system.
+* Download and install the binary executable for your platform from the 
+[GitHub release page](https://github.com/Originate/exosphere/releases)
+* Rename the executable to `exo`
+* Check and update your `PATH` environmental variable to include its install location, if required
 
-### macOS
+## Managing multiple versions of Exosphere
 
-To install the SDK manually:
+A common pattern for managing executables on a system is to install the executable with
+the version in the filename and symlinking the desired command name to the executable.
+Using this pattern, old versions are still available in your `PATH`. In the event that you need
+to run an old version of Exosphere it will still be available as `exo-X.Y.Z` unless it is explicitly 
+removed.
```

Comment:
Is that really a thing right now? Is there a use case for this? Afaik we don't provide support for older versions since we don't have stable releases yet. I would prefer that everybody always uses the latest version. Once we reach a the second stable release (2.0), we can support older stable releases (1.x) longer.